src/main/java/jiachen/duke/Duke.java

```diff
@@ -0,0 +1,338 @@
+package jiachen.duke;
+
+import javafx.application.Application;
+import javafx.application.Platform;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+import java.util.ArrayList;
+
+/**
+ * The enum Command.
+ */
+enum Command {
+  /**
+   * Exit command command.
+   */
+  EXIT_COMMAND,
```

Comment:
The comments to all the enum command is self-explainable. It really needed, could add more description. Overall, like what ZiYingLi has commented, the code is very well-structured and organised. Well-done